pkg/ingester/metrics.go

```diff
@@ -563,6 +585,28 @@ func newTSDBMetrics(r prometheus.Registerer) *tsdbMetrics {
 			"cortex_ingester_memory_series_removed_total",
 			"The total number of series that were removed per user.",
 			[]string{"user"}, nil),
+
+		ephemeralHeadTruncateFail: prometheus.NewDesc(
+			"cortex_ingester_ephemeral_head_truncations_failed_total",
+			"Total number of TSDB head truncations that failed (for ephemeral storage).",
+			nil, nil),
+		ephemeralHeadTruncateTotal: prometheus.NewDesc(
+			"cortex_ingester_ephemeral_head_truncations_total",
+			"Total number of TSDB head truncations attempted (for ephemeral storage).",
+			nil, nil),
+		ephemeralHeadGcDuration: prometheus.NewDesc(
+			"cortex_ingester_ephemeral_head_gc_duration_seconds",
+			"Runtime of garbage collection in the TSDB head (for ephemeral storage).",
```

Comment:
I'd keep this a simple direct sentence, but that's just personal preference.

```suggestion
			"Total number of TSDB head truncations that failed for ephemeral storage.",
			nil, nil),
		ephemeralHeadTruncateTotal: prometheus.NewDesc(
			"cortex_ingester_ephemeral_head_truncations_total",
			"Total number of TSDB head truncations attempted for ephemeral storage.",
			nil, nil),
		ephemeralHeadGcDuration: prometheus.NewDesc(
			"cortex_ingester_ephemeral_head_gc_duration_seconds",
			"Runtime of garbage collection in the TSDB head for ephemeral storage.",
```

Reply:
Done in https://github.com/grafana/mimir/pull/3922/commits/d9f55b01d2a35d1feefa2300321fb10150c5b239